config/routes.rb

```diff
@@ -1,4 +1,8 @@
 Rails.application.routes.draw do
   devise_for :users
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  resources :tweets
```

Comment:
Gotcha, will limit tweet to [:index] and add onto it as each route is needed. Thanks! :balloon: